test/Microsoft.AspNet.Mvc.Core.Test/ModelBinding/FormCollectionModelBinderTest.cs

```diff
@@ -77,8 +77,8 @@ public class FormCollectionModelBinderTest
 
             // Assert
             Assert.NotNull(result);
-            Assert.IsType(typeof(FormCollection), result.Model);
-            Assert.Empty((FormCollection)result.Model);
+            Assert.IsType(typeof(IFormCollection), result.Model);
+            Assert.Empty((IFormCollection)result.Model);
```

Comment:
``` c#
var collection = Assert.IsAssignableFrom<IFormCollection>(result.Model);
Assert.Empty(collection);
```

Not sure how test would pass as-is since `IsType()` verifies the exact type "(and not a derived type)".